src/Components/Button/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+
+export const Button = ({ clickHandler, buttonText, isDisabled }) => (
+  <button
+    className={styles.Button}
+    disabled={isDisabled}
+    onClick={clickHandler}
+  >
+    {buttonText}
```

Comment:
one day you'd like to put there some loaders/icons inside your button so having `{children}` there I think is more suitable